### DISCUSSION TITLE
Consiglio per apertura garage

### DISCUSSION THREAD

**Discussion by Paoloc66** [indomus/forum#2675] [GENERALE - Discussioni generiche]
Buongiorno a tutti avrei questa esigenza e chiedo consiglio:
In casa ho HA su miniPC.
Vorrei comandare l'apertura (già automatica) del garage da HA. In garage c'è un classico pulsante che apre la serranda.
In garage non prende il wifi di casa, ma è collegata alla stessa linea elettrica. 
Secondo voi qual'è la soluzione più economica e fattibile?
Pensavo un lan powerline per portare la lan in garage e poi? Per attivare il pulsante?
Grazie

**Comment by Scialla** [comment]
Non è certo che il powerline funzioni da casa al garage, in ogni caso non mettere ciabatte o triple in mezzo, prova collegando direttamente alle prese.
Se hai un pulsante che apre la porta, per aprire solamente ti basta un contatto pulito che ritorna a off dopo un secondo, ad esempio uno shelly1, ma non hai lo stato dell'apertura/chiusura. Per avere lo stato ti devi ingegnare con dei fine corsa.

Valuta anche di usare qualcosa in RF che ha la portata maggiore. Io ad esempio lo uso per le porte dei box e delle cantine, vedi se trovi online con contatto pulito in RF, così non avresti bisogno dei powerline

**Comment by Paoloc66** [comment]
Oggi ho fatto un controllo e purtroppo le powerline non funzionano. Dovrò usare qualcosa in RF come dicevi. Hai qualcosa da consigliarmi? Tu cosa usi?

**Reply by Scialla** [reply]
Io uso sia un vecchio sonoff BridgeRF tasmotato (quello nuovo è più ostico) che un vecchio Broadlink 3 RM Pro.
Non ho attuatori a contatto pulito da consigliarti in quanto uso RF solo per sensori porta.
Devi cercare tu su AliExpress
